src/libs/ReportUtils.ts

```diff
@@ -2690,6 +2688,9 @@ function canDeleteReportAction(
             if (!isEmptyObject(report) && (isMoneyRequestReport(report) || isInvoiceReport(report))) {
                 return canDeleteTransaction(report) && isCardTransactionCanBeDeleted;
             }
+            if (isTrackExpenseAction(reportAction)) {
+                return isCardTransactionCanBeDeleted;
```

Comment:
`isCardTransactionCanBeDeleted` how is this var different than `canCardTransactionBeDeleted`? I think we should only use the latter